noxfile.py

```diff
@@ -127,9 +133,12 @@ def get_sphinx_build_command(kind):
         return [
             "sphinx-build",
             "-W",
-            "-c", "docs/html",  # see note above
-            "-d", "docs/build/doctrees/" + kind,
-            "-b", kind,
+            "-c",
+            "docs/html",  # see note above
+            "-d",
+            "docs/build/doctrees/" + kind,
```

Comment:
Looks like separating related CLI args from their values harms maintainability here

Reply:
Oh yea, this is a good catch. I'll fix this.